### ISSUE TITLE
Dependency issue on Atom >= v1.0.12.

### ISSUE THREAD

**Issue by ghost** [romgrk/termrk#63]
Hello

It seems that in Atom v1.0.12 something is now going wrong with the `pty.js` dependency. The 'Incompatible Packages' logo is shown and Atom indicates that `Termrk` is incompatible with the following error:

```
Some installed packages could not be loaded because they contain native modules that were compiled for an earlier version of Atom.

pty.js@0.2.13 – Module version mismatch. Expected 44, got 43.
```

I suspect this has something to do with the upgrade to Chromium 43 and io.js 2.3.1, but I'm not sure.

Thanks in advance

**Comment by ghost** [comment]
This can be closed; it turns out this was an Atom bug, which is now resolved in Atom v1.0.15, where you could not request the package to be rebuilt from the incompatibility screen. After rebuilding it there and restarting, it now works again.

Sorry for bothering you with this.

**Comment by romgrk** [comment]
No problem!
